src/applications/personalization/dashboard-2/components/apply-for-benefits/ApplicationInProgress.jsx

```diff
@@ -3,48 +3,118 @@ import PropTypes from 'prop-types';
 
 import { recordDashboardClick } from '~/applications/personalization/dashboard/helpers';
 
+/**
+ * Returns a copy of the passed-in string with the first letter capitalized
+ *
+ * @param {string} input - word to capitalize
+ * @returns {string} - the input string with the first letter capitalized
+ */
+const capitalizeFirstLetter = input => {
+  const capitalizedFirstLetter = input[0].toUpperCase();
```

Comment:
This works, but we could also accomplish this with css using `text-transform: capitalize`

Reply:
🤦 

Oh wait, that will capitalize all the words (ie dumb title casing)